### PR TITLE
[merged] .redhat-ci.yml: new file

### PR DESCRIPTION
**Pull request by giuseppe**
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by giuseppe** [comment]
@rh-atomic-bot r+ 7125a60

**Comment by ashcrow** [comment]
@giuseppe just curious, it doesn't look like the ``.redhat-ci.yml`` file is set up to do anything in this PR. What's the purpose of adding it right now?

**Comment by giuseppe** [comment]
@ashcrow I am working right now to add better tests like: https://github.com/giuseppe/syscontainers-lint
I've submitted this place holder file just to get things in place and check that it is correctly integrated.

**Comment by cgwalters** [comment]
@rh-atomic-bot r+ 7125a60

**Comment by giuseppe** [comment]
@rh-atomic-bot retry

**Comment by jlebon** [comment]
You'll need the same `branches` blurb as for the bwrap-oci one.

**Comment by giuseppe** [comment]
@rh-atomic-bot retry

**Comment by giuseppe** [comment]
@rh-atomic-bot retry

**Comment by giuseppe** [comment]
@rh-atomic-bot retry

File: .redhat-ci.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+container:
+    image: fedora:24
+
+packages:
+    - python
+    - findutils
+    - git
+
+build: false
```
Comment (jlebon) [inline .redhat-ci.yml]: That's the default behaviour, so you shouldn't need this.

**Comment by jlebon** [comment]
Ahh, the PR has new commits, so homu needs another r+ before it will test it. :)

**Comment by giuseppe** [comment]
@rh-atomic-bot r+

@jlebon amended, let's see if it works now :)

**Pull request closed without merging**